apps/emqx_dashboard/src/emqx_dashboard_monitor.erl

```diff
@@ -319,10 +319,10 @@ do_sample(Node, Time) when is_integer(Time) ->
 
 do_sample_local(Time) ->
     MS = ets:fun2ms(fun(#emqx_monit{time = T} = A) when T >= Time -> A end),
-    FromDB = ets:select(?TAB, MS),
+    FromDB = lists:keysort(#emqx_monit.time, ets:select(?TAB, MS)),
```

Comment:
sorry, i was maybe wrong. 
there is actually no need to sort after all ? it turns into a map anyway.

Reply:
d'oh, you're right.   
sorted in the wrong spot :see_no_evil:

Reply:
I already added a sort in `ts_list/1`.